src/actions/users.js

```diff
@@ -197,6 +198,19 @@ function completeLogin(data) {
             },
         ]), getState);
 
+        const roles = new Set();
```

Comment:
I keep seeing this pattern over and over again, can we create an util function for this?